### DISCUSSION TITLE
ASMB Round 3 - No staking event found in firefly

### DISCUSSION THREAD

**Discussion by AlexT010101** [iotaledger/firefly#4127] [Help & Support] [answered]
Hi,

i have no option to stake my Gi to get ASMB. New Firefly Version 1.6.5 installed.

**Comment by AlexT010101** [comment, marked as answer]
Fixed -> Network Settings - Nodes. Save, re-open Wallet -> fixed

**Comment by David120712** [comment]
Hi,

i have also no option to stake my Mi to get ASMB. 
I have new Firefly Version 1.6.5 installed.

**Reply by David120712** [reply]
I re-try the same way again now is fixed
